content/03.methods.md

```diff
@@ -130,6 +130,28 @@ We used RSEM [@doi:10/cwg8n5] v1.3.1 for both FPKM and TPM transcript- and gene-
 We also added a second method of quantification using kallisto [@doi:10.1038/nbt.3519] v0.43.1.
 This method differs in that it uses pseudoaligments using fastq reads directly to the aforementioned GENCODE 27 reference.
 
+### Gene Expression Matrices with Unique HUGO Symbols
+Algorithms that perform gene set enrichment, molecular subtyping or immune-profiling require a RNA-seq gene expression matrix as input, with HUGO gene symbols as row names and sample names as column names. This is however not straight-forward to achieve as there are a small proportion of gene symbols which are mapped to multiple Ensembl gene identifiers. In Gencode v27, there are 212 gene symbols mapped to 1866 Ensembl gene identifiers. 
```

Comment:
```suggestion
Algorithms that perform gene set enrichment, molecular subtyping, or immune-profiling, for example, require an RNA-seq gene expression matrix as input, with HUGO gene symbols as row names and sample names as column names. 
This is not straight-forward, as there are a small proportion of gene symbols that map to multiple Ensembl gene identifiers (in Gencode v27, 212 gene symbols map to 1866 Ensembl gene identifiers). 
```

@komalsrathi thanks for this! Starting to review, but can you separate each sentence on its own line and remove the line spacing between paragraphs? Made a few minor edits above to condense.